SDL_GameControllerGetStringForAxis.mediawiki

```diff
@@ -28,6 +27,51 @@ Returns a string for the given axis, or NULL if an invalid axis is
 specified. The string returned is of the format used by
 [[SDL_GameController]] mapping strings.
 
+== Code Examples == 
+<!-- #Example1 Start -->
+<syntaxhighlight lang='c++'>
+#include <iostream>
+#include <cstdlib>
+
+#include <SDL.h>
+
+int main(int argc, char* argv[])
+{
+    if(SDL_Init(SDL_INIT_GAMECONTROLLER) < 0)
+    {
+        std::cerr << "Error while initializing SDL2 library : " << SDL_GetError() << std::endl;
+        return EXIT_FAILURE;
+    }
+
+    SDL_Event event;
+
+    bool running = true;
+    while(running)
+    {
+        while(SDL_PollEvent(&event) > 0)
+        {
+            if(event.type == SDL_QUIT)
+                running = false;
+
+            std::string axisName = "";
+            int axisValue = 0;
+            if(event.type == SDL_CONTROLLERAXISMOTION)
+            {
+                axisName = SDL_GameControllerGetStringForAxis((SDL_GameControllerAxis) event.caxis.axis);
+                axisValue = event.caxis.value;
+                std::cout << "Axis used : " << axisName << "\tAxis value : " << axisValue << std::endl;
```

Comment:
You should use SDL_Log() instead of std::cout. This will work on mobile devices and Windows gui applications.

Reply:
Ok, I'll change this. Thx